pkg/reconciler/v1alpha1/configuration/configuration_test.go

```diff
@@ -654,3 +665,7 @@ func TestIsRevisionStale(t *testing.T) {
 		})
 	}
 }
+
+func WithoutConfigMetadataGenerationLabel(rev *v1alpha1.Revision) {
```

Comment:
Why not in `functional.go`?

Reply:
1. The related tests & helpers will be dropped in 0.6.
2. This helper is only used in one place.